packages/app-store/googlevideo/components/InstallAppButton.tsx

```diff
@@ -1,64 +1,20 @@
-import { useState } from "react";
-
 import type { InstallAppButtonProps } from "@calcom/app-store/types";
-import useApp from "@calcom/lib/hooks/useApp";
-import { useLocale } from "@calcom/lib/hooks/useLocale";
-import type { DialogProps } from "@calcom/ui";
-import { Button } from "@calcom/ui";
-import { Dialog, DialogClose, DialogContent, DialogFooter } from "@calcom/ui";
 
 import useAddAppMutation from "../../_utils/useAddAppMutation";
 
 export default function InstallAppButton(props: InstallAppButtonProps) {
```

Comment:
Revert this back to a normal install button

Reply:
We can delete this file itself. If InstallAppButton.tsx file doesn't exist, default InstallAppButton component is used located at https://github.com/calcom/cal.com/blob/670520c282032816555b9cb309fe2d566ae7b754/packages/app-store/components.tsx#L30